src/blockfilter.cpp

```diff
@@ -243,6 +244,7 @@ const std::string& ListBlockFilterTypes()
     static std::once_flag flag;
     std::call_once(flag, []() {
             std::stringstream ret;
+            ret.imbue(std::locale::classic());
```

Comment:
Would it be possible to add a `static std::stringstream ClassicStringStream() { std::stringstream s; s.imbue(); return s; }` to our str util library, so that call sites have a lower LOC-overhead?